src/lib/holocene/icon/paths.ts

```diff
@@ -219,23 +219,23 @@ const globeIcon = {
   paths: [
     {
       d: 'M6 11C8.7614 11 11 8.7614 11 6C11 3.23857 8.7614 1 6 1C3.23857 1 1 3.23857 1 6C1 8.7614 3.23857 11 6 11Z',
-      stroke: '',
+      stroke: 'currentColor',
```

Comment:
I think "currentcolor" is technically supposed to be all lowercase, but if this works, nbd.

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword

Reply:
Yeah it works, but good call on checking.